utility/src/main/java/org/oppia/android/util/profile/BUILD.bazel

```diff
@@ -0,0 +1,12 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
```

Comment:
Add some documentation to explain what the package does.

Reply:
I added a comment declaring what the package does, at the top of the file. Is that enough?